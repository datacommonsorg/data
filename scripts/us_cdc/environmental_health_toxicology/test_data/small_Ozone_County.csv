Year,Month,Day,statefips,countyfips,O3_max_pred,O3_med_pred,O3_mean_pred,O3_pop_pred
2001,JAN,1,01,1,31.939058,31.691988,31.6808585833333,31.6712262223708
2001,JAN,1,01,3,33.646855,33.170271,32.9947749354838,32.9350697957369
2001,JAN,1,01,5,34.288917,34.068507,34.0779537777777,34.0866306011945
2001,JAN,1,01,7,30.349767,30.036093,29.93175575,29.991733426271
2001,JAN,1,01,9,26.472655,25.776595,25.8575708888888,25.8724720383971
2001,JAN,1,01,11,33.607696,33.252916,33.3046376666666,33.1696366148066
2001,JAN,1,01,13,33.385263,32.992025,32.9911305555555,32.9857658772139
2001,JAN,1,01,15,29.553653,28.873731,28.7909083870967,28.7797639908072
2001,JAN,1,01,17,32.269248,32.032674,31.963699,31.9937919549028
2001,JAN,1,01,19,28.613335,28.4270775,28.3153388333333,28.292539279195
2001,JAN,1,01,21,31.132989,30.743824,30.6450868888888,30.572113369223
2001,JAN,1,01,23,33.950801,33.7021405,33.686666,33.6776395035716
2001,JAN,1,01,25,33.887748,33.648907,33.6081631111111,33.5661207622421
2001,JAN,1,01,27,30.688122,30.1181705,30.11563425,30.1605159267872
2001,JAN,1,01,29,30.855293,30.120469,30.20309275,30.2681012427865
2001,JAN,1,01,31,34.196473,33.9589405,33.9934574285714,33.9955487814326
2001,JAN,1,01,33,27.533899,25.682831,25.8164068571428,25.7950421543323
2001,JAN,1,01,35,33.752266,33.303454,33.3758202,33.4532000275929
2001,JAN,1,01,37,31.138993,30.106737,30.3926723333333,30.3744259493023
