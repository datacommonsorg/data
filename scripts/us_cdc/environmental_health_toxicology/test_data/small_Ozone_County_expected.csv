Year,Month,Day,statefips,countyfips,O3_max_pred,O3_med_pred,O3_mean_pred,O3_pop_pred,date,dcid
2001,1,1,1,12001,31.939058,31.691988,31.680859,31.671226,2001-01-01,geoId/12001
2001,1,1,1,12003,33.646855,33.170271,32.994775,32.935070,2001-01-01,geoId/12003
2001,1,1,1,12005,34.288917,34.068507,34.077954,34.086631,2001-01-01,geoId/12005
2001,1,1,1,12007,30.349767,30.036093,29.931756,29.991733,2001-01-01,geoId/12007
2001,1,1,1,12009,26.472655,25.776595,25.857571,25.872472,2001-01-01,geoId/12009
2001,1,1,1,120011,33.607696,33.252916,33.304638,33.169637,2001-01-01,geoId/120011
2001,1,1,1,120013,33.385263,32.992025,32.991131,32.985766,2001-01-01,geoId/120013
2001,1,1,1,120015,29.553653,28.873731,28.790908,28.779764,2001-01-01,geoId/120015
2001,1,1,1,120017,32.269248,32.032674,31.963699,31.993792,2001-01-01,geoId/120017
2001,1,1,1,120019,28.613335,28.427077,28.315339,28.292539,2001-01-01,geoId/120019
2001,1,1,1,120021,31.132989,30.743824,30.645087,30.572113,2001-01-01,geoId/120021
2001,1,1,1,120023,33.950801,33.702140,33.686666,33.677640,2001-01-01,geoId/120023
2001,1,1,1,120025,33.887748,33.648907,33.608163,33.566121,2001-01-01,geoId/120025
2001,1,1,1,120027,30.688122,30.118171,30.115634,30.160516,2001-01-01,geoId/120027
2001,1,1,1,120029,30.855293,30.120469,30.203093,30.268101,2001-01-01,geoId/120029
2001,1,1,1,120031,34.196473,33.958940,33.993457,33.995549,2001-01-01,geoId/120031
2001,1,1,1,120033,27.533899,25.682831,25.816407,25.795042,2001-01-01,geoId/120033
2001,1,1,1,120035,33.752266,33.303454,33.375820,33.453200,2001-01-01,geoId/120035
2001,1,1,1,120037,31.138993,30.106737,30.392672,30.374426,2001-01-01,geoId/120037
